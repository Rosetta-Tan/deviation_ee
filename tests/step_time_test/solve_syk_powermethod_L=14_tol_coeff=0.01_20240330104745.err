Traceback (most recent call last):
  File "/n/home01/ytan/deviation_ee/syk/solve_syk_powermethod.py", line 125, in <module>
    main()
  File "/n/home01/ytan/deviation_ee/syk/solve_syk_powermethod.py", line 108, in main
    v.save(os.path.join(vec_dir,f'v0_L={L}_seed={seed}_tol_coeff={tol_coeff}'))
UnboundLocalError: local variable 'v' referenced before assignment
