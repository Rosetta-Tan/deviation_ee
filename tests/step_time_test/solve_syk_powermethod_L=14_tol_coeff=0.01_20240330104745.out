load Hamiltonian, L=14, seed=10, to be solved at tol_coeff 0.01; time elapsed: 0:00:00.003841
construct the two operators involved in power method, L=14, seed=10; time elapsed: 0:00:00.006330
